test/mocks/mac/StyleBench/urls.js

```diff
@@ -1,8 +1,8 @@
-  "https://treeherder.mozilla.org/api/project/mozilla-central/performance/data/?framework=10&interval=1209600&signature_id=1742701&signature_id=1742702&signature_id=1742703&signature_id=1742704&signature_id=1742705&signature_id=1742706&signature_id=1742707&signature_id=1742708&signature_id=1742709&signature_id=1742710&signature_id=1742711&signature_id=1742712&signature_id=1742713&signature_id=1742714&signature_id=1742715&signature_id=1742716&signature_id=1742717&signature_id=1742718&signature_id=1742719&signature_id=1742720&signature_id=1742721&signature_id=1742722&signature_id=1742723&signature_id=1742724&signature_id=1742725&signature_id=1742726&signature_id=1742727&signature_id=1742728&signature_id=1742729"
+  "https://treeherder.mozilla.org/api/project/mozilla-central/performance/data/?framework=10&signature_id=1707552&signature_id=1707553&signature_id=1707554&signature_id=1707555&signature_id=1707556&signature_id=1707557&signature_id=1707558&signature_id=1707559&signature_id=1707560&signature_id=1707561&signature_id=1707562&signature_id=1707563&signature_id=1707564&signature_id=1707565&signature_id=1707566&signature_id=1707567&signature_id=1707568&signature_id=1707569&signature_id=1707570&signature_id=1707571&signature_id=1707572&signature_id=1707573&signature_id=1707574&signature_id=1707575&signature_id=1707576&signature_id=1707577&signature_id=1707578&signature_id=1707579&signature_id=1707580&signature_id=1707581&signature_id=1707582&signature_id=1707583&signature_id=1707584&signature_id=1707585&signature_id=1707586&signature_id=1707587&signature_id=1707588&signature_id=1707589&signature_id=1707590&signature_id=1707591&signature_id=1707592&signature_id=1707593&signature_id=1707594&signature_id=1707595&signature_id=1707596&signature_id=1707597&signature_id=1707598&signature_id=1707599&signature_id=1707600&signature_id=1707601&signature_id=1707602&signature_id=1707603&signature_id=1742353&signature_id=1742354&signature_id=1742355&signature_id=1742356&signature_id=1742357&signature_id=1742358&signature_id=1742359&signature_id=1742360&signature_id=1742361&signature_id=1742362&signature_id=1742363&signature_id=1742364&signature_id=1742365&signature_id=1742366&signature_id=1742367&signature_id=1742368&signature_id=1742369&signature_id=1742370&signature_id=1742371&signature_id=1742372&signature_id=1742373&signature_id=1742374&signature_id=1742375&signature_id=1742376&signature_id=1742377&signature_id=1742378&signature_id=1742379&signature_id=1742380&signature_id=1742381&signature_id=1742382&signature_id=1742383&signature_id=1742384&signature_id=1742385&signature_id=1742386&signature_id=1742387&signature_id=1742388&signature_id=1742389&signature_id=1742390&signature_id=1742391&signature_id=1742392&signature_id=1742393&signature_id=1742394&signature_id=1742395&signature_id=1742396&signature_id=1742397&signature_id=1742398&signature_id=1742399&signature_id=1742400",
```

Comment:
It'd be worth mentioning in that TH bug you filed that the default behavior should be to return data for ALL signatures unless specific signatures are specified. So `https://treeherder.mozilla.org/api/project/mozilla-central/performance/data/?framework=10` should give you the same data as what you're doing now. Currently `signature_id` is a required parameter so that would need to be changed in that API.

Reply:
Also, I got an application error when trying this url above. The Perfherder  frontend had to batch these requests to 150 signatures at a time before it switched to the new [endpoint] (https://github.com/sarah-clements/treeherder/blob/dab3785ace60a7d916e1a13c2abe161173817904/ui/perfherder/helpers.js#L279). I think that's because of a query parameter limit.

Reply:
Although I see your code hasn't changed... if this worked with your app then maybe it's fine?